nss-cohort-37/Bangazon-Site-Green-Monkeys
# Mac home page

# Description
When you view the home page you will see a welcome message as well as a list of the latest 20 items

Fixes #18 

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Log in and view the home page. Confirm that you see the latest products. 
Click `read more` and you will be taken to that product's details view. 



# Checklist:
